agents/plugins/mk_zypper

```diff
@@ -9,39 +9,58 @@
 
 # Disable unused variable error (needed to keep track of version)
 # shellcheck disable=SC2034
-CMK_VERSION="2.1.0i1"
+CMK_VERSION="2.0.0p17"
 
 #SuSE-release is deprecated and was removed with SLE 15. os-release should be used for the new versions.
 
 RELEASEFILE="/etc/SuSE-release"
+STATEFILE="/var/lib/check_mk_agent/zypper.state"
+SPOOLFILE="/var/lib/check_mk_agent/spool/90000_mk_zypper"
```

Comment:
Maybe this should be dynamic? It should be "allowed max age from below" + "cache interval" + "some grace period", right?